api/Element.json

```diff
@@ -3153,28 +3260,59 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/requestFullscreen",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true,
+              "alternative_name": "webkitRequestFullscreen"
             },
             "chrome": {
-              "version_added": null
+              "version_added": true,
```

Comment:
This was in [version 15](https://www.chromestatus.com/features/5259513871466496).